docs/site/tutorials/a_swift_tour.ipynb

```diff
@@ -178,7 +188,7 @@
         "let width = 94\n",
         "label + String(width)"
       ],
-      "execution_count": 0,
+      "execution_count": 5,
```

Comment:
Did you want to clear execution outputs before checking this in? Or is it intentional to have the outputs preserved?

Reply:
Typically, we don't save outputs. But Swift notebooks must save outputs because our tool that imports to tensorflow.org doesn't yet support swift notebook execution.
For that, we need a docker image with a Swift kernel that we can run `nbconvert` within.